app/blog/(components)/YoutubeEmbed.tsx

```diff
@@ -0,0 +1,22 @@
+"use-client"
```

Comment:
invalid directive, did you mean 'use client'?

Reply:
meany to delete that anyway haha.